### ISSUE TITLE
Prologus added

### ISSUE THREAD

**Issue by CharlotteFeidicker** [scta-texts/bcg6es#3]
Dear @jeffreycwitt , as you know, I am currently working on the fifth distinctio of the collectiones. In this part of the text, Wilhelm reffered to other parts of the collectiones, for example to the prologus. I wondered if it might be interesting to link these references, therefore one could jump directly to the reference (to the prologus in this case). To provide this opportunity, I created the xml-structured text-file for the prologus in the folder with the prologus id. I copied the <teiHeader/> of the fifth dist. and pasted it into the prologus-file. That followed, I changed the <title/> in the title-statement and edition-statement to prologus, the date of creation to today's date and the div-id to the prologus-id. Furthermore, I inserted paragraph-breaks <p/> where the text showed them and added ids with the id-pattern to them. 

The Id for the <head/> is missing because I did not know the rules for the pattern. 

A little note to pay attention to: I corrected the transcription only for the fifth disctinctio, so the prologus is not yet corrected. 

Is it ok when I add parts of the collectiones like I descriped?

**Comment by CharlotteFeidicker** [comment]
It seems like it is not always exactly clear to what part of the prologus Wilhelm refers to. Maybe we have to think about this again.

**Comment by CharlotteFeidicker** [comment]
The result would look like this: See in the first picture the reference in the fifth distinctio: 
![Nec expectandum est](https://user-images.githubusercontent.com/56366898/113848467-ca501080-9798-11eb-8d2d-8fc7032fb272.png)

And in the second picture the part in the prologus that was refered to: 
![Nec expectandum est_2](https://user-images.githubusercontent.com/56366898/113848482-ce7c2e00-9798-11eb-9ae0-bae0f93ca47c.png)

Actually, in this moment, I am not sure if the source-Attribute might be a better or a least a good additional opiton to refer to the prologus within the SCTA-reading-room.

**Comment by jeffreycwitt** [comment]
Instead of using `@corresp` you should use `ref@target` and `quote@source`. Basically you should treat it just like any other reference or quotation. The distinction between cross reference and references no longer really applies in the SCTA world. In a way every reference/quotation is a cross reference. 

Here is how I would encode your example:

```xml
<ref xml:id="bcg6es-d1e110-Rd1e219" target="http://scta.info/resource/bcg6es-d1e69-d1e429">
  Prologo <quote type="incipit" source="http://scta.info/resource/bcg6es-d1e69-d1e429" synch="2-4">Nec expectandum est</quote>
</ref>
```



I understand Wilhelm's reference to be a particular paragraph in the prologue
Within the reference, he uses a "quote@type=incipit" to help the reader finding the target reference. So I've marked this as a special type of quote, whose source is the same paragraph, plus the word range, 2-4.

**Comment by CharlotteFeidicker** [comment]
Thanks for your explanation and the encoded example! This is a great solution. 
Just to be sure: Should I never use the `ref@corresp` in any reference and always the `ref@target` in every reference ?

**Comment by CharlotteFeidicker** [comment]
Well, it seems like I got confused for any reason that I can't remember. I read the LombardPress-Guidelines again and there is written that `ref@target` should be used. So I'ill go with that instead of `ref@corresp `.

**Comment by CharlotteFeidicker** [comment]
> 
> 
> Well, it seems like I got confused for any reason that I can't remember. I read the LombardPress-Guidelines again and there is written that `ref@target` should be used. So I'ill go with that instead of `ref@corresp `.

This is not true, have a look to : https://github.com/scta/scta-rdf-schema/blob/master/docs/proposals/quote-ref-relations.md
